apps/chat-e2e/src/tests/promptMaximumNameLength.test.ts

```diff
@@ -0,0 +1,177 @@
+import dialTest from '@/src/core/dialFixtures';
+import {
+  ExpectedConstants,
+  ExpectedMessages,
+  MenuOptions,
+} from '@/src/testData';
+import { Overflow, Styles } from '@/src/ui/domData';
+import { expect } from '@playwright/test';
+
+dialTest(
+  'Prompt name consists of a maximum of 160 symbols.\n' +
+    'Long prompt name is cut in the panel.\n' +
+    'Prompt folder name consists of a maximum of 160 symbols',
+  async ({
+    dialHomePage,
+    promptData,
+    dataInjector,
+    prompts,
+    promptDropdownMenu,
+    promptModalDialog,
+    errorToastAssertion,
+    promptAssertion,
+    setTestIds,
+    promptBar,
+    folderPrompts,
+  }) => {
+    setTestIds('EPMRTC-3171', 'EPMRTC-958', 'EPMRTC-3168');
+    const prompt = promptData.prepareDefaultPrompt();
+    await dataInjector.createPrompts([prompt]);
+    const longName =
+      'Lorem ipsum dolor sit amett consectetur adipiscing elit. Nullam ultricies ipsum nullaa nec viverra lectus rutrum id. Sed volutpat ante ac fringilla turpis duis!ABC';
+    const expectedName = longName.substring(
+      0,
+      ExpectedConstants.maxEntityNameLength,
+    );
+    const nameUnder160Symbols =
+      'This prompt is renamed to very long-long-long name to see how the system cuts the name';
+
+    await dialTest.step(
+      'Create a prompt and enter text longer than 160 symbols',
+      async () => {
+        await dialHomePage.openHomePage();
+        await dialHomePage.waitForPageLoaded({
+          isNewConversationVisible: true,
+        });
+        await promptBar.createNewPrompt();
+        await promptModalDialog.setField(promptModalDialog.name, longName);
+        await promptModalDialog.setField(
+          promptModalDialog.prompt,
+          ExpectedConstants.newPromptTitle(1),
+        );
+      },
+    );
+
+    await dialTest.step('Save the prompt', async () => {
+      await promptModalDialog.saveButton.click();
+    });
+
+    await dialTest.step(
+      'Verify the prompt name is cut to 160 symbols and no error toast is shown',
+      async () => {
+        await promptAssertion.assertEntityState(
+          { name: expectedName },
+          'visible',
+        );
+        await errorToastAssertion.assertToastIsHidden();
+      },
+    );
+
+    await dialTest.step('Rename the prompt to a long name', async () => {
+      await prompts.openEntityDropdownMenu(expectedName);
+      await promptDropdownMenu.selectMenuOption(MenuOptions.edit);
+      await promptModalDialog.setField(
+        promptModalDialog.name,
+        nameUnder160Symbols,
+      );
+      // Wait for the API request to update the prompt name
+      await promptModalDialog.updatePromptDetailsWithButton(
+        nameUnder160Symbols,
+        prompt.description || '',
```

Comment:
I think it would be better to do empty/undefined check inside `PromptModalDialog.fillPromptDetails()` method

Reply:
done